spec/jobs/performance_report_job_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+
+class DummyWrongTestFirst
```

Comment:
I like this method of creating Dummy classes - nice and simple way to do things 👍  - one small thing is indentation is 4 spaces here I usually in ruby its 2.